MFVNeutralino/test/K0Tracks/effplot.C

```diff
@@ -46,8 +53,10 @@ void effplot()
   rp->GetLowerRefGraph()->SetLineColor(kBlue);
   rp->GetLowerRefGraph()->SetMinimum(0.85);
   rp->GetLowerRefGraph()->SetMaximum(1.15);
+  /*rp->GetLowerRefGraph()->SetMinimum(-1);
+  rp->GetLowerRefGraph()->SetMaximum(6);*/
   rp->GetLowerRefYaxis()->SetTitle("data/MC");
-  //rp->GetUpperPad().SetLogy();
+  //rp->GetUpperPad()->SetLogy();
```

Comment:
@duquale it would be fine to delete all of the things that you commented out / aren't using anymore! Especially the directory paths that aren't being used.